TYPO3.Neos/Resources/Private/Styles/Media/_Media.scss

```diff
@@ -643,6 +595,27 @@
 						max-width: 100%;
 					}
 				}
+
+				.neos-img-label {
+					padding: 0 0 1rem .5rem;
+					position: relative;
+					font-size: 12px;
+
+					.neos-caption {
+						padding: 0 0 0 .5rem;
+						margin-right: 2rem;
```

Comment:
`rem` shouldn't be used in general, all sizes are defined in pixels and usually relative to `$defaultMargin`, `$relatedMargin`, `$tightMargin` or `$unit`